frontend/public/components/graphs/query-browser.tsx

```diff
@@ -37,6 +37,12 @@ export const graphColors = [
   '#1f77b4', '#ff7f0e', '#2ca02c', '#d62728', '#9467bd', '#8c564b', '#e377c2', '#7f7f7f', '#bcbd22', '#17becf',
 ];
 
+const NoQueryMessage = () => <div className="text-center text-muted">Enter a query in the box below to explore the metrics gathered for this cluster</div>;
+
+const Error = ({error}) => <div className="alert alert-danger">
+  <span className="pficon pficon-error-circle-o" aria-hidden="true"></span>{_.get(error, 'json.error', error.message)}
```

Comment:
nit: prefer self-closing tags

(We should add eslint or prettier rules)

```suggestion
  <span className="pficon pficon-error-circle-o" aria-hidden="true" />{_.get(error, 'json.error', error.message)}
```

Reply:
Fixed as part of https://github.com/openshift/console/pull/1767